dashboard/app/views/plc/tasks/new.html.haml

```diff
@@ -2,7 +2,4 @@
 
 = render 'form'
 
```

Comment:
If you got to this page with a learning_module_id param, might as well let the content creator go back to that module